README.md

```diff
@@ -393,6 +419,12 @@ Terra groups are managed by SAM. These commands are utility wrappers around the
 
 The `enterprise-pilot-testers` group is used for managing access to the default WSM spend profile.
```

Comment:
This is an operational detail that belongs in a private repo or document, not in the open source one.

Reply:
Done